### PR TITLE
Update sorting on changelog to include always-descending-by-timestamp…

### PR DESCRIPTION
**Pull request by kelfink**
… MadeTo column sorting

#### Which User story does this relate to (link)?
- https://osi-cwds.atlassian.net/browse/COG-1260

#### Brief feature description
- Sorting update:  when the Made To column is selected for sorting, there is a secondary sort on the time stamp, always descending (as in the Made By column)

### CODE REVIEW

File: app/javascript/views/userDetail/ChangeLog.test.js [hunk 1]
```diff
@@ -163,17 +163,57 @@ describe('ChangeLog', () => {
...
+        event: { admin_name: 'apple' },
+        timestamp: '2050-01-01 12:00:00',
+      }
+      const rowB = {
+        event: { admin_name: 'apple' },
+        timestamp: '2000-01-01 12:00:00',
+      }
+      expect(rowA.event.admin_name).toEqual('apple')
+      expect(rowB.event.admin_name).toEqual('apple')
+      expect(sortByName(rowA, rowB, false)).toEqual(-1)
```
Comment (RamuKammagani-osi) [inline app/javascript/views/userDetail/ChangeLog.test.js]: shouldn't this be ```expect(sortByMadeTo(rowA, rowB, false)).toEqual(-1)```???

Comment (kelfink) [reply app/javascript/views/userDetail/ChangeLog.test.js]: Yes!  And several other updates...

File: app/javascript/views/userDetail/ChangeLog.test.js [hunk 2]
```diff
@@ -163,17 +163,56 @@ describe('ChangeLog', () => {
...
+    it('sorts by event user_name', () => {
+      const rowA = { event: { user_name: 'apple' } }
+      const rowB = { event: { user_name: 'banana' } }
+      expect(sortByMadeTo(rowA, rowB)).toEqual(-1)
+      expect(sortByMadeTo(rowB, rowA)).toEqual(1)
+    })
+
+    it('breaks a tie with timestamp', () => {
+      const rowA = {
+        event: { admin_name: 'apple' },
```
Comment (RamuKammagani-osi) [inline app/javascript/views/userDetail/ChangeLog.test.js]: ```user_name``` instead of ```admin_name``` ??

**Comment by RamuKammagani-osi** [comment]
looks good!

**Pull request merged**